src/resources/js/tickets.js

```diff
@@ -637,16 +637,21 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 
 		$tribe_tickets.trigger( 'pre-save-ticket.tribe', e );
 
-		var $orders = $base_panel.find( '.tribe-ticket-field-order' );
+		var $ticketID = $edit_panel.find( '#ticket_id' ).val();
+		var $editParent = $base_panel.find( `[data-ticket-type-id="${$ticketID}"]` );
```

Comment:
Only use the `$` on jQuery objects. In this case, `ticketID` is a string.
```suggestion
		var ticketID = $edit_panel.find( '#ticket_id' ).val();
		var $editParent = $base_panel.find( `[data-ticket-type-id="${ticketID}"]` );
```

Reply:
yes nice catch